docs/workflow/building/libraries/README.md

```diff
@@ -227,3 +221,17 @@ If you are working on Windows, and use Visual Studio, you can open individual li
 For more details about running tests inside Visual Studio, [go here](../../testing/visualstudio.md).
 
 For more about running tests, read the [running tests](../../testing/libraries/testing.md) document.
+
+## Build packages
+To build a library's package, simply invoke `dotnet pack` on the src project after you successfully built the .NETCoreApp vertical from root:
```

Comment:
@ViktorHofer is there a `./build.sh` subset for building library packages?

Reply:
@tmds yes there is. libs.packages -- however you would need to pass down an extra parameter. 

`./build.sh libs.packages -allConfigurations`. IIRC you will also need a prebuilt .NETCoreApp vertical from root.